joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstCreator.scala

```diff
@@ -55,7 +55,7 @@ class AstCreator(filename: String, phpAst: PhpFile, global: Global) extends AstC
 
     val globalTypeDecl = NewTypeDecl()
       .name(namespaceBlock.name)
-      .fullName(namespaceBlock.fullName)
+      .fullName(namespaceBlock.name)
```

Comment:
This is intended?

Reply:
It is. The namespaceBlock fullname contains too much information (the filename, for example). Whether this is correct is a different discussion, but for the purposes of this PR the name is actually what we want.